### PR TITLE
Python 3 support

### PR DESCRIPTION
**Pull request by ericof**

### CODE REVIEW

File: src/collective/jsonmigrator/tests/test_utils.py [hunk 1]
```diff
@@ -55,14 +54,14 @@ def test_remove_first_bar(self):
     def test_remove_first_bar_converte(self):
```
Comment (wesleybl) [inline src/collective/jsonmigrator/tests/test_utils.py:54]: This test can be removed. It doesn't make sense anymore.

File: src/collective/jsonmigrator/blueprints/properties.py [hunk 2]
```diff
@@ -1,26 +1,19 @@
-# -*- coding: utf-8 -*-
```
Comment (wesleybl) [inline src/collective/jsonmigrator/blueprints/properties.py:1]: The code below can be removed:

```python
  if not isinstance(pvalue, str):
      # BBB: Python 2
      pvalue = pvalue.encode("utf-8")
```

File: src/collective/jsonmigrator/blueprints/configure.zcml [hunk 3]
```diff
@@ -1,83 +1,84 @@
...
+      component=".statistics.Statistics"
       />
 
   <utility
-      component=".workflowhistory.WorkflowHistory"
       name="collective.jsonmigrator.workflowhistory"
+      component=".workflowhistory.WorkflowHistory"
       />
 
   <!-- BBB: Plone 4.3 doesn't come with plone.app.multilingual -->
```
Comment (wesleybl) [inline src/collective/jsonmigrator/blueprints/configure.zcml:77]:
```suggestion
```

**Review by wesleybl — CHANGES_REQUESTED** [review]

**Comment by wesleybl** [comment]
@ericof I think you missed some comments that github hid. "5 hidden conversations".

**Comment by ericof** [comment]
@wesleybl Reviewed all comments

**Review by wesleybl — APPROVED** [review]
LGTM

**Comment by wesleybl** [comment]
@ericof thanks for your work!

**Pull request merged**